modules/nf-core/sourmash/compare/meta.yml

```diff
@@ -0,0 +1,63 @@
+name: "sourmash_compare"
+description: Compare many FracMinHash signatures generated by sourmash sketch.
+keywords:
+  - compare
+  - FracMinHash sketch
+  - containment
+  - sourmash
+  - metagenomics
+  - genomics
+  - kmer
+tools:
+  - "sourmash":
+      description: "Compute and compare FracMinHash signatures for DNA and protein data sets."
+      homepage: "https://sourmash.readthedocs.io/"
+      documentation: "https://sourmash.readthedocs.io/"
+      tool_dev_url: "https://github.com/sourmash-bio/sourmash"
+      doi: "10.21105/joss.00027"
+      licence: "['BSD-3-clause']"
+
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test', single_end:false ]
+  - signatures:
+      type: file
+      description: Files containing signatures (hash sketches) of samples
+      pattern: "*.{sig}"
+  - save_numpy_matrix:
+      type: boolean
+      description: |
+        If true, output will contain a (dis)similarity matrix numpy binary format
```

Comment:
Maybe also doucment here at least one of save_numpy_matrix/save_csv is required?